### PR TITLE
Revising default Wordpress file name. Updating documentation. Updating change log.

### PR DESCRIPTION
**Pull request by dl-unleashed-technologies**

### CODE REVIEW

**Review by colinodell — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,15 @@ Updates should follow the [Keep a CHANGELOG](https://keepachangelog.com/) princi
 
 ## [Unreleased][unreleased]
 
+## [0.1.5] - 2021-09-20
```
Comment (colinodell) [inline CHANGELOG.md:7]: The release date needs to be changed.

Or, if you aren't quite ready to release this immediately, you should temporarily remove this line plus the changes at the bottom of this until you're just about to make that release.

File: cms/wp/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# WordPress Deployer Config
+
+## Example WordPress Configuration
+
+The following will configure Deployer to deploy `git@repository.example:foo/wp`
+to `foo.dev.example`, `foo.staging.example` & `production.example`. It will
+expect that `example-staging` is a [defined host](https://www.ssh.com/academy/ssh/config#format-of-ssh-client-config-file-ssh_config).
```
Comment (colinodell) [inline cms/wp/README.md:7]: I don't see that repository or any of the host names above (including `example-staging`) referenced anywhere in this section - but maybe I'm missing something?

File: CHANGELOG.md (hunk 1 above)
Comment (dl-unleashed-technologies) [reply CHANGELOG.md:7]: Noted! This release will go out tomorrow.

File: cms/wp/README.md (hunk 2 above)
Comment (dl-unleashed-technologies) [reply cms/wp/README.md]: The documentation has been updated in a follow-up commit. I am marking this conversation as resolved, in favor of any follow-up discussion that may need to occur given the new changes.

**Pull request merged**